spec/requests/api/v1/contacts/show_spec.rb

```diff
@@ -24,11 +24,11 @@
       end
 
       it "returns 200 and provides the appropriate contact details" do
-        get api_v1_user_contacts_path(@user.id, @contact.id), headers: { "Authorization" => "Bearer #{@token}" }, as: :json
+        get api_v1_user_contact_path(@user.id, @contact.id), headers: { "Authorization" => "Bearer #{@token}" }, as: :json
 
         expect(response).to be_successful
 
-        json = JSON.parse(response.body, symbolize_names: true)[:data].first
```

Comment:
The `.first` should've been a giveaway to our original reviwers that something was amiss here. Great work discovering the issue with the specs.